src/test/java/hudson/plugins/ec2/util/AmazonEC2FactoryMockImpl.java

```diff
@@ -40,21 +50,98 @@ public static AmazonEC2 createAmazonEC2Mock() {
      *            nullable
      * @return mocked AmazonEC2
      */
-    public static AmazonEC2 createAmazonEC2Mock(@Nullable Answer defaultAnswer) {
-        AmazonEC2 mock;
+    public static AmazonEC2Client createAmazonEC2Mock(@Nullable Answer defaultAnswer) {
+        AmazonEC2Client mock;
         if (defaultAnswer != null) {
-            mock = Mockito.mock(AmazonEC2.class, defaultAnswer);
+            mock = Mockito.mock(AmazonEC2Client.class, defaultAnswer);
         } else {
-            mock = Mockito.mock(AmazonEC2.class);
+            mock = Mockito.mock(AmazonEC2Client.class);
```

Comment:
You can use static imports for `Mockito` methods.

Reply:
Added static import now, but I actually prefer not to have them. With `Mockito.mock` it makes it obvious that it's Mockito and not e.g. PowerMock.

Similarly to `assertEquals`; are we talking AssertJ, Junit, something else? I just prefer the explicitness :)